api/ask_astro/settings.py

```diff
@@ -20,3 +20,5 @@
 CONVERSATIONAL_RETRIEVAL_LOAD_QA_CHAIN_DEPLOYMENT_NAME = os.environ.get(
     "CONVERSATIONAL_RETRIEVAL_LOAD_QA_CHAIN_DEPLOYMENT_NAME", "gpt-4-32k"
 )
+
+SERVICE_MAINTENANCE_BANNER_STATUS = os.environ.get("SERVICE_MAINTENANCE_BANNER", "False") == "True"
```

Comment:
Should we rename it as `SHOW_SERVICE_MAINTENANCE_BANNER`? The name looks a bit confusing to me. Also should we use lower to compare?

Reply:
yes, `SHOW_SERVICE_MAINTENANCE_BANNER ` makes me more sense